pkgs/development/python-modules/sendgrid/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, pyyaml
+, python-http-client
+, six
+, pytest
+, mock
+, codecov
+, coverage
+, fetchurl
+}:
+ 
+buildPythonPackage rec {
+  pname = "sendgrid";
+  version = "6.1.1"; 
+ 
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5ad98598c13b8cf5545d20f6fb6ecf967a08b48e3e175affabd82cfc71522d01";
+  };
+
+ checkPhase = ''
+  test --ignore=tests/test_app.py
+ '';
+ 
+ propagatedBuildInputs = [ flask pyyaml six pytest mock codecov coverage python-http-client];
```

since this is quite a few packages, please do one per line
```suggestion
 checkPhase = ''
  test --ignore=tests/test_app.py
 '';
 
 propagatedBuildInputs = [
   flask 
   pyyaml
   ...
 ];
```